test/test.php

```diff
@@ -46,12 +46,9 @@ public function isOk() {
   'intl'
 );
 
-// remove IoCube Loader from required extensions if php version is greater than/equal to 7.1
-if (version_compare(phpversion(), '7.1.0') >= 0) {
-    $key = array_search('IonCube Loader', $required_extensions);
-    if (false !== $key) {
-        unset($required_extensions[$key]);
-    }
+$key = array_search('IonCube Loader', $required_extensions);
+if (false !== $key) {
+    unset($required_extensions[$key]);
 }
```

Comment:
I don't think we should be removing IonCube from testing in all versions now that we have it installed in all versions.

Reply:
The code _removed_ the IonCube check for PHP versions greater than 7.1.  The IonCube loader is now being checked on the 7.1 image as it's defined in `$required_extensions`.

Reply:
Ah good spot, I didn't mean to do this.